perfscale_regerssion_ci/kubeburner-object-templates/pause-config.yml

```diff
@@ -0,0 +1,16 @@
+jobs:
```

Comment:
are these files being used in either of the scripts below?

Reply:
Now this is used by the new added run_workload.sh to run kube-burner run.sh